CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ FEATURES:
 
 ## 1.18.1 (August 26, 2024)
 
+NOTES:
```

Comment:
Doing the changelog adjustment manually as it wont get picked up by our changelog generation process.